integreat_cms/static/src/js/forms/tinymce-init.ts

```diff
@@ -184,6 +184,28 @@ window.addEventListener("load", () => {
             },
             readonly: !!tinymceConfig.getAttribute("data-readonly"),
             init_instance_callback: (editor: Editor) => {
+                editor.shortcuts.add("alt+|+1", "Add pin icon", () => {
+                    addIcon(editor, tinymceConfig, "pin");
```

Comment:
```suggestion
                    let icon = tinymceConfig.getAttribute(`data-pin-icon-src`);
                    editor.insertContent(`<img src="${icon}" style="width:15px; height:15px">`);
```

The function "addIcon" adds an icon into the icon choice (the list which expands after clicking on "icon" menu) but not in the text.
Inserting an icon into the text can be achived, for example, like the suggestion above.

Could you apply the same change for the other icons too? You can also write a function for that if you find it not cool to repeat those code lines.